linera-views/src/backends/memory.rs

```diff
@@ -361,6 +361,9 @@ impl AdminKeyValueStore for MemoryStore {
         let mut memory_stores = MEMORY_STORES
             .lock()
             .expect("MEMORY_STORES lock should not be poisoned");
+        if memory_stores.sync_exists(namespace) {
```

Comment:
Soon (with @bart-linera 's work) DB initialization will be idempotent because it will only be about creating blobs.